lumigator/python/mzai/backend/backend/services/jobs.py

```diff
@@ -64,7 +71,7 @@ def __init__(
         self.job_repo = job_repo
         self.result_repo = result_repo
         self.ray_client = ray_client
-        self.data_service = data_service
+        self._dataset_service = data_service
```

Comment:
```suggestion
        self._dataset_service = dataset_service
```
Please also update this in the method parameters for consistency